Content.Shared/GameObjects/EntitySystems/SharedMoverSystem.cs

```diff
@@ -156,7 +156,7 @@ private static void HandleDirChange(ICommonSession? session, Direction dir, usho
             {
                 foreach (var comp in owner.GetAllComponents<IRelayMoveInput>())
                 {
-                    comp.MoveInputPressed(session);
+                    comp.MoveInputPressed(session!);
```

Comment:
this should be handled properly.

Reply:
It's impossible for `session` to be `null` here, but just to avoid using an `!` I added an extra check to the if statement to shut the compiler up.